test/integration/scale/scale_test.go

```diff
@@ -58,7 +53,7 @@ func TestScaleSubresources(t *testing.T) {
 	})
 	defer tearDown()
 
-	resourceLists, err := clientSet.Discovery().ServerResources()
+	_, resourceLists, err := clientSet.Discovery().ServerGroupsAndResources()
```

Comment:
what seems to be the problem with using `clientSet.Discovery().ServerResources()`?

Reply:
`ServerResources` is deprecated https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/discovery/discovery_client.go#L211